dali/python/backend_impl.cc

```diff
@@ -1201,9 +1195,19 @@ void ExposeTensorList(py::module &m) {
       may be viewed as a tensor of shape `(N, H, W, C)`.
       )code")
     .def("copy_to_external",
-        [](TensorList<GPUBackend> &t, py::object p, py::object cuda_stream,
+        [](TensorList<GPUBackend> &tl, py::object p, py::object cuda_stream,
            bool non_blocking, bool use_copy_kernel) {
-          CopyToExternalImplGPU(t, p, cuda_stream, non_blocking, use_copy_kernel);
+          CUDAStreamLease lease;
+          AccessOrder copy_order;
+          int device = tl.device_id();
+          if (!cuda_stream.is_none()) {
+            cudaStream_t stream = static_cast<cudaStream_t>(ctypes_void_ptr(cuda_stream));
+            copy_order = AccessOrder(stream, device);
+          } else {
+            lease = CUDAStreamPool::instance().Get(device);
+            copy_order = AccessOrder(lease, device);
+          }
+          CopyToExternalImplGPU(tl, p, cuda_stream, non_blocking, use_copy_kernel);
```

Comment:
Doesn't CopyToExternalImplGPU already have similar code obtaining some stream? Why is it repeated here? `copy_order` seems unused.

Reply:
Mhh, leftover. Removed